AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -32,6 +32,8 @@ reddit.com#$#body.scroll-disabled { overflow: visible!important; position: stati
 !+ NOT_PLATFORM(ios, ext_android_cb)
 reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPage"] > div[class="NavFrame"] > div[class="XPromoPopup active"]
 !
+baseball.yahoo.co.jp,sports.yahoo.co.jp###RSAB
+baseball.yahoo.co.jp,sports.yahoo.co.jp##.sn-modInterstitial
```

Comment:
I think it's better to move this rule to popups.txt

Reply:
Done both.